ueimporter/job.py

```diff
@@ -46,38 +26,79 @@ def create_jobs(changes, plastic_repo, source_root_path, pretend, logger):
 
         for i in range(0, len(per_file_changes) - 1):
             change = per_file_changes[i]
-            next_change = per_file_changes[i+1]
-            if type(change) != git.Delete or type(next_change) != git.Add:
+            if not change:
                 continue
 
-            logger.log('Replacing Del + Add with Move')
-            per_file_changes[i] = git.Move(change.filename,
-                                           next_change.filename)
-            per_file_changes[i + 1] = None
-
-        all_per_file_changes += [c for c in per_file_changes if c]
+            next_change = per_file_changes[i+1]
+            if type(change) == git.Delete and type(next_change) == git.Add:
+                move = git.Move(change.filename, next_change.filename)
+
+                logger.log('Replacing')
+                logger.indent()
+                for change_to_log in [change, next_change]:
+                    for line in str(change_to_log).split('\n'):
+                        logger.log(line)
+                logger.deindent()
+                logger.log('With')
+                logger.indent()
+                for line in str(move).split('\n'):
+                    logger.log(line)
+                logger.deindent()
+
+                change = move
+                per_file_changes[i+1] = None
+
+            job_changes = change_type_to_changes.get(type(change))
+            assert job_changes != None
+            job_changes.append(change)
+
+    logger.deindent()
+
+    # Convert Move to Modify if move appears to already happened
+    # in the target repo
+    logger.log('Finding moves that have already been applied in target.')
+    logger.indent()
+    moves_already_existing_in_target = [ \
```

Comment:
This backslash is superfluous, right?

Reply:
I guess it is, the interpreter should be able to use the brackets to figure out that the line continues.

I will remove it